dali/operators/image/crop/bbox_crop.cc

```diff
@@ -354,7 +354,19 @@ The values are:
         R"code(If set to True, an extra output will be returned, containing
 the original indices of the bounding boxes that passed the centroid filter and are present in
 the output bounding boxes.)code",
-        false);
+        false)
+    .AddOptionalArg<float>("bbox_prune",
+        R"code(Controls how bboxes are pruned from the ROI. Valid values of `bbox_prune` are `-1.f` 
```

Comment:
How about instead of using `-1` as an off value, call `ArgumentDefined` to learn if the argument was used at all?